src/components/filters/time-widget.js

```diff
@@ -85,37 +73,21 @@ const TopSectionWrapper = styled.div`
 
       .item-selector__dropdown__value {
         color: ${props =>
-        props.hoverColor
-          ? props.theme[props.hoverColor]
-          : props.theme.textColorHl};
+          props.hoverColor
+            ? props.theme[props.hoverColor]
+            : props.theme.textColorHl};
       }
     }
   }
-`;
 
-/* eslint-disable no-unused-vars */
-const Tabs = styled.div`
-  padding-right: 76px;
-`;
+  .animation-control__speed-control {
+    margin-right: -12px;
 
-const Tab = styled.div`
-  border-bottom: 1px solid
-    ${props => (props.active ? props.theme.textColorHl : 'transparent')};
-  color: ${props =>
-  props.active ? props.theme.textColorHl : props.theme.labelColor};
-  display: inline-block;
-  font-size: 12px;
-  height: 24px;
-  margin-right: 4px;
-  text-align: center;
-  width: 24px;
-  line-height: 24px;
-
-  :hover {
-    cursor: pointer;
+    .animation-control__speed-slider {
+      right: calc(0% - 48px);
```

Comment:
constant

Reply:
don't think we have to use constant in css everywhere, this is only so that the slider will align with the edge